test/components/Collapse.spec.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import { expect } from 'chai';
 import sinon from 'sinon';
-import {
-  mount,
-} from 'enzyme';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
 import util from '../../src/util';
 import Collapse from '../../src/components/Collapse';
 
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
This part could be done in a own test setup file if I am not mistaking

Reply:
Ok 👍